libraries/botbuilder-ai/src/qnaMakerDialog.ts

```diff
@@ -194,7 +194,6 @@ export class QnAMakerDialog extends WaterfallDialog implements QnAMakerDialogCon
      * @param strictFilters (Optional) QnA Maker metadata with which to filter or boost queries to the knowledge base; or null to apply none.
      * @param dialogId (Optional) Id of the created dialog. Default is 'QnAMakerDialog'.
      */
-    public constructor();
     public constructor(
```

Comment:
Every parameter is optional, so there is no need to explicitly list an overload.